docs/Deployed_Contracts/Ethereum/sepolia.md

```diff
@@ -0,0 +1,23 @@
+# Sepolia
```

Comment:
I don't think this file needs the `Ethereum` We can just move this file int `Deployed_Contracts`

Also it would be good to figure out how to not display in the app "Deployed_Contract" and instead display "Deployed Contracts"